dev/prepare_db.pql

```diff
@@ -4,41 +4,73 @@ func run_sql(code) {
     force_eval( SQL( nulltype, code ))
 }
 
+func drop_table(t) {
+    run_sql("DROP TABLE IF EXISTS " + t)
+}
+
+func create_indices(tbl) {
+    tbl.add_index("id", true)
+    tbl.add_index("timestamp")
+    tbl.add_index(["id", "timestamp"])
```

Comment:
Ideally these would be created as part of the table definition instead of built afterward. It's nicer if that's defined together

Reply:
Generally I agree, but I'm not sure what's the best way to do it, syntax-wise.

Especially on (id, timestamp).

Also, does it mean that indexes are copied when you copy a table?

This just seemed like the simplest, most robust solution.